lib/dialects/freebsd/Mksrc

```diff
@@ -12,14 +12,21 @@
 #
 # $Id: Mksrc,v 1.5 2008/04/15 13:31:47 abe Exp $
 
+mksrc() {
```

Comment:
Didn't another similar change cause this file to be shabanged as `/bin/bash` on another dialect?